improver_tests/nbhood/square_kernel/test_SquareNeighbourhood.py

```diff
@@ -86,8 +86,7 @@ def test_basic(self):
                          [4., 8., 11., 15., 19.],
                          [5., 10., 14., 19., 24.]])
         cube = set_up_cube(
-            zero_point_indices=((0, 0, 2, 2),), num_time_points=1,
-            num_grid_points=5)
+            zero_point_indices=((0, 0, 2, 2),), num_grid_points=5)
```

Comment:
Yes, true.  While you're at it could you redirect the import statement for `set_up_cube` to its actual source, please.  (I don't quite know how it's working as it is...)

Reply:
Do you mean `from ..nbhood.test_NeighbourhoodProcessing import set_up_cube` to `from improver_tests.nbhood.nbhood.test_BaseNeighbourhoodProcessing import
    set_up_cube`